### PR TITLE
Kp/hot fix

### PR DESCRIPTION
**Pull request by kprokofi**
WIP
- [x] fix inference with a small number of images
- [x] Fix scaling in score extraction for AM softmax
- [x] update templates (scale, loss for multilabel, learning rate)

### CODE REVIEW

**Comment by kprokofi** [comment]
@sovrasov , ready fo review

File: configs/ote_custom_classification/mobilenet_v3_large_1/main_model.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ train:
   ema:
     enable: True
     ema_decay: 0.999
-  mix_precision: False
+  mix_precision: True
```
Comment (sovrasov) [inline configs/ote_custom_classification/mobilenet_v3_large_1/main_model.yaml]: I think it's better to revert this change, since NCT team doesn't expect that.

File: torchreid/engine/image/am_softmax.py [hunk 2]
```diff
@@ -336,7 +323,7 @@ def _single_model_losses(self, model, train_records, imgs, obj_ids, n_iter, mode
                                                     lam=self.lam, iteration=n_iter, scale=self.scales[model_name])
                 avg_acc += metrics.accuracy(trg_logits, trg_obj_ids)[0].item()
                 loss_summary['main_{}/{}'.format(trg_id, model_name)] = main_loss.item()
-
+
```
Comment (sovrasov) [inline torchreid/engine/image/am_softmax.py]: Is it a trailing whitespace?

File: torchreid/engine/image/multilabel.py [hunk 3]
```diff
@@ -44,7 +45,13 @@ def __init__(self, datamanager, models, optimizers, schedulers, use_gpu, save_al
         if not isinstance(num_classes, (list, tuple)):
             num_classes = [num_classes]
         self.num_classes = num_classes
-        self.scale = s
+        self.scales = dict()
```
Comment (sovrasov) [inline torchreid/engine/image/multilabel.py]: May be this code should be extracted to the Engine class, since it's used by more than one subclass (something like collect_scales() method)

File: configs/ote_custom_classification/mobilenet_v3_large_1/main_model.yaml (hunk 1 above)
Comment (kprokofi) [reply configs/ote_custom_classification/mobilenet_v3_large_1/main_model.yaml]: Yes, I overlooked it

File: torchreid/engine/image/multilabel.py (hunk 3 above)
Comment (kprokofi) [reply torchreid/engine/image/multilabel.py]: agree

**Review by sovrasov — APPROVED** [review]
